src/HomeWork09/Main.java

```diff
@@ -0,0 +1,28 @@
+package HomeWork09;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println();
+        System.out.println("Задание 1 ");
+
+        String firstName = "Ivan ";
+        String middleName = "Ivanovich ";
+        String lastName = "Ivanov ";
+        String fullName = lastName + firstName + middleName;
```

Comment:
Лучше пробелы хранить не в самой строке, а в сборке полного имени
```suggestion
        String fullName = lastName + " " +  firstName + " "+ middleName;
```